hitbox-tarantool/src/init.lua

```diff
@@ -0,0 +1,34 @@
+local fiber = require("fiber")
+local log = require("log")
+
+box.cfg({})
+
+local space_name = ...
+
+box.schema.space.create(space_name, { if_not_exists = true })
```

Comment:
Let's create a lua interface for reusing it in sharded implementation in the future.

Reply:
done